php/libraries/SinglePointLogin.class.inc

```diff
@@ -48,119 +48,6 @@ class SinglePointLogin
     var $_passwordExpired = false;
 
     /**
-     * Dumps the login form
-     *
-     * @return void
-     * @access public
-     */
-    function showLoginScreen()
-    {
-        $tpl_data = array();
-        if (isset($_POST['username'])) {
-            $tpl_data['username'] = $_POST['username'];
-        } else {
-            $tpl_data['username'] = '';
-        }
-
-        $tpl_data['currentyear'] = date('Y');
-        $tpl_data['version']     = file_get_contents(__DIR__ . "/../../VERSION");
-
-        // remove logout from request
-        if (isset($_SERVER['QUERY_STRING'])) {
-            parse_str($_SERVER['QUERY_STRING'], $params);
-            unset($params['logout']);
-        } else {
-            $params = array();
-        }
-
-        $config =& NDB_Config::singleton();
-
-        $tpl_data['page_title']    = $config->getSetting('title');
-        $tpl_data['css']           = $config->getSetting('css');
-        $tpl_data['action']        = $_SERVER['PHP_SELF']
-            . '?' . http_build_query($params);
-        $tpl_data['error_message'] = $this->_lastError;
-
-        $study_links = $config->getExternalLinks('Studylinks');
-        foreach ($study_links as $label => $url) {
-            $WindowName = md5($url);
-            $tpl_data['studylinks'][] = array(
-                                         'url'        => $url,
-                                         'label'      => $label,
-                                         'windowName' => $WindowName,
-                                        );
-        }
-
-        $study_title       = $config->getSetting('title');
-        $study_description = $config->getSetting('StudyDescription');
-        $tpl_data['study_title']       = $study_title;
-        $tpl_data['study_description'] = html_entity_decode($study_description);
-        try {
-            $tpl_data['study_logo'] = $config->getSetting('studylogo');
```

Comment:
study_logo does not show anymore on login
![screenshot from 2017-10-19 12-07-15](https://user-images.githubusercontent.com/15268287/31781289-15575250-b4c6-11e7-8f81-3eec2c45a321.png)

Reply:
where are you referring to? It seems like it's showing up properly on my sandbox..

Reply:
in the config table, this is the value for the study_logo.

images/logo.png

Reply:
also note text in panel headers

![screenshot from 2017-10-19 15-36-34](https://user-images.githubusercontent.com/15268287/31790410-5f865458-b4e3-11e7-9632-3a7c64f1ed00.png)

Reply:
Quickly looking at the code I can see that the study logo path is never passed to the template form_login.tpl... it was deleted in the transfer I guess

Reply:
FIXED